fixedwidth/src/main/java/org/apache/metamodel/fixedwidth/FixedWidthReader.java

```diff
@@ -85,6 +92,15 @@ public FixedWidthReader(InputStream stream, String charsetName, int[] valueWidth
         _expectedLineLength = expectedLineLength;
     }
 
+    private void initReader() {
+        try {
+            InputStreamReader inputStreamReader = new InputStreamReader(_stream, _charsetName);
```

Comment:
This was the missing part in the previous version, _charsetName was not used anywhere.